<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingMeal</class>
 <widget class="QWidget" name="SettingMeal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_meallist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>441</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>菜品列表</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_addmeal" native="true">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>80</y>
     <width>281</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 239, 222)</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜名字:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line2_mealname_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜类型:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comb2_type">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜价格:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin2_price">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_add">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增菜品</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>待开发功能区</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改菜信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新增菜品</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_mealdetail" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>281</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(170, 170, 255)</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜名字:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>菜价格:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comb1_type">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin1_price">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line1_mealid">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line1_mealname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点击修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_delete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点击删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
